VAMobile/documentation/docs/QA/Automation/Detox Best Practices.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Detox Best Practices
+sidebar_position: 2
+---
+
+# Detox Best Practices
+
+This is a set of best practices around creating and maintaining the detox tests.
+
+### Creating/updating detox tests for a new feature
+
+For any new feature work detox work needs to either be:
+- Added to the acceptance criteria and done on a per ticket basis
+- Done in a new ticket at the end of the feature that encompasses all the work completed (If a new ticket needs to be made please follow the Engineering best practices for ticket creation.)
+
+The following must be done for all new feature detox work no matter whether you are updating an existing detox test or creating a new one. 
+
+- Ensure that the test encompasses any new changes to the manual RC script and test steps where possible
+	- Anything that happens outside of the app itself can't be automated with Detox.  In the past this has included stuff like opening and selecting an attachment in messages etc.
+- Inform QA what has been automated.  This allows QA to:
+	- Move the correct manual cases to the automated folder/split a case if needed in testRail
+	- Confirm that all test runs for the new automated script can be recorded in TestRail (preferably automatically, but at least manually)
+- Ensure the new/updating script provides artifacts (where necessary) for success and failure
+- Ensure that the new test has been added to the array/dictionary in the Navigation.e2e and AvailabilityFramework.e2e tests (if needed). How to add new tests can be found [here](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA/Automation/AddingNewFeatures)
```

Comment:
Can this be a relative link?

Reply:
Yes it can and I've changed it to be one